client/scripts/Components/__tests__/Datefields.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { createRenderer } from 'react-test-renderer/shallow';
+import moment from 'moment';
+
+import DateFields from '../DateFields';
+
+const DEFAULT_PROPS = {
+  stateDate: moment.unix(1526311584),
```

Comment:
Since the Travis boxes are set to a different internal timezone we have to force a particular time using a unix timestamp.  Good thing to keep in mind for next time!